car:
  bio: Something
  body_rgb:
  - 255
  - 255
  - 255
  body_style: car01
  car_name: Triton-ROSer
  country: US
  font_size: 30
  guid: some_random_string
  racer_name: Triton AI
default_connection: local
lidar:
  deg_inc: 2
  enabled: true
  max_range: 50.0
  numSweepsLevels: 4
  rot_x: 0
  degAngDelta: 0
local_connection:
  artificial_latency: 0
  host: host.docker.internal
  port: 9091
  scene_name: warren
remote_connection:
  artificial_latency: 0
  host: 127.0.0.1
  port: 9091
  scene_name: warren
  